0-bootstrap/main.tf

```
@@ -47,7 +49,7 @@ module "seed_bootstrap" {
   project_prefix                 = var.project_prefix
 
   # Remove after github.com/terraform-google-modules/terraform-google-bootstrap/issues/160
-  depends_on = [google_folder.bootstrap]
+  depends_on = [google_folder.bootstrap, module.google_groups.required_groups]
```

```suggestion
  depends_on = [google_folder.bootstrap, module.google_groups[0].required_groups]
```

but i think it could be only

```suggestion
  depends_on = [google_folder.bootstrap, module.google_groups]
```